src/main/java/org/example/Extractor.java

```diff
@@ -1,26 +1,67 @@
 package org.example;
 
+import org.jsoup.Connection;
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
 import org.jsoup.nodes.Element;
-
+import org.jsoup.select.Elements;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Extractor {
-    public List<Chair> getChairs(String url){
+    public List<Chair> getChairs(String url) throws IOException {
+
 
         String role, name, email;
         List<Chair> chairs = new ArrayList<>();
-        //todo
+
+        Connection connection = Jsoup.connect("https://www.ualberta.ca/computing-science/faculty-and-staff/index.html");
+        Document document = connection.get();
+
+        Elements Chairs = document.getElementsByTag("tr");
+        for (Element Chair : Chairs) {
+
+            Elements td0 = Chair.getElementsByAttribute("Style");
+            Elements td1 = Chair.getElementsByTag("strong");
+            Elements td2 = Chair.getElementsByAttribute("href");
+            String Role = td0.text().trim();
```

Comment:
Please do not use a capital letter at the beginning of the variable name and instead of calling it "Role" please make use of a variable called "role" which is declared at the beginning of this method.

Reply:
resolved